### PR TITLE
Switch CI to the fedora-python Tox GitHub action

### PR DESCRIPTION
**Pull request by encukou**
Fixes: https://github.com/fedora-python/tox-current-env/issues/25

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
```
Comment (hroncok) [inline .github/workflows/main.yaml:7]: IMO Any branch should run the tests, not just master.

```suggestion
  push:
  pull_request:
```

**Comment by hroncok** [comment]
Out of scope notes for me:

 - add Python 3.9
 - set up a cron job so the test for tox master makes sense
 - replace tox 3.5 with 3.13 (Fedora 31)

**Comment by encukou** [comment]
> can you please reopen this PR from a branch in this repo

Apparently the new PR is not needed, pushing the commit to a branch in this repo was enough.

**Comment by encukou** [comment]
Thanks!

**Pull request merged**